Adjust some default settings (default shipped .conf files) for openSUSE and openSUSE based appliances.
Updating databases during installation may cause problems, thus allow to install system as is.
Calamares packages is no longer needed after installation.

diff --git a/src/modules/packages/packages.conf b/src/modules/packages/packages.conf
--- a/src/modules/packages/packages.conf
+++ b/src/modules/packages/packages.conf
@@ -12,7 +12,7 @@
 #  - entropy     - Sabayon package manager
 #  - dummy       - Dummy manager, only logs
 #
-backend: dummy
+backend: zypp
 
 # Often package installation needs an internet connection.
 # Since you may allow system installation without a connection
@@ -28,7 +28,7 @@
 # list of packages and dependencies); this is done only if there
 # is an internet connection. Set *update_db* to true to do so.
 skip_if_no_internet: false
-update_db: true
+update_db: false
 
 #
 # List of maps with package operations such as install or remove.
@@ -125,11 +125,5 @@
 # of packages, this can lead to a considerable time savings.
 #
 operations:
-  - install:
-    - vi
-    - wget
-    - binutils
-  - remove:
-    - vi
-    - wget
-    - binutils
+  - try_remove:
+    - calamares
